### ISSUE TITLE
Getting "unknown file attribute: h" running beep

### ISSUE THREAD

**Issue by ftraxler-tgm** [spkr-beep/beep#9]

**Comment by ndim** [comment]
That sounds very interesting, as I have no idea what that could be.

Could you please elaborate a bit? To begin with

  * What version of beep? (`beep --version` and maybe the operating system's package version)

  * What kind of operating system and which version?

  * What `beep` command line?

  * What is the output when you add `--verbose` to that `beep` command line?

We will see where we need to go from there.

**Comment by ndim** [comment]
@ftraxler-tgm At this time, I suspect you are running some zsh incompatible shell script called `beep` under the zsh shell, but I will need your feedback to help confirming or disproving this hypothesis and understanding what is actually happening here.

Looking forward to your feedback.

**Comment by ndim** [comment]
@ftraxler-tgm are you still there? Have you resolved the problem already?

**Comment by ftraxler-tgm** [comment]
Yes I am currently using zsh but I have not tested it with bash since.

**Comment by ndim** [comment]
Ah, thank you very much for the reply.

Now the question is
  * what does that script actually do (`cat "$(which beep)"`) and
  * who wrote and shipped that script

because this project does not ship any shell scripts called `beep`.

I suspect that script to be a manually created script left over from an old manual installation, or from some distribution package.
